src/site/index.rst

```diff
@@ -136,6 +136,7 @@ Labs
     lab10
     lab11
 
+
```

Comment:
Why tho

Reply:
There were two lines after each grouping in there. This one was the weird one.